hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestOMMultiTenantManagerImpl.java

```diff
@@ -65,18 +62,22 @@ public void setUp() throws IOException {
     OMMetadataManager omMetadataManager = new OmMetadataManagerImpl(conf);
 
     final String bucketNamespaceName = tenantName;
-
-    // Note: policyNames is initialized with an empty list here.
-    //  Expand if needed.
-    final OmDBTenantState omDBTenantInfo = new OmDBTenantState(
-        tenantName, bucketNamespaceName, new ArrayList<>());
-
-    omMetadataManager.getTenantStateTable().put(tenantName, omDBTenantInfo);
+    final String bucketNamespacePolicyName =
+        OMMultiTenantManager.getDefaultBucketNamespacePolicyName(tenantName);
+    final String bucketPolicyName =
+        OMMultiTenantManager.getDefaultBucketNamespacePolicyName(tenantName);
```

Comment:
Same as above, this looks like it should be `OMMultiTenantManager.getDefaultBucketPolicyName(tenantName);`